packages/vscode-extension/package.nls.json

```diff
@@ -400,6 +400,8 @@
     "teamstoolkit.tutorials.connectApi.label": "Connect to an API",
     "teamstoolkit.tutorials.addAzureSql.label": "Integrate with Azure SQL Database",
     "teamstoolkit.tutorials.addAzureSql.detail": "A platform as a service (PaaS) database engine to serve as your Teams applications backend.",
+    "teamstoolkit.tutorials.addTab.label": "Configure Tab capability",
+    "teamstoolkit.tutorials.addTab.detail": "Teams identity-aware webpages embedded in Microsoft Teams",
```

Comment:
Desc: Teams-aware webpages embedded in Microsoft Teams.

Reply:
Updated.